packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -12,64 +13,19 @@ import {
     routes,
 } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
-import { connect } from 'Stores/connect';
-import { TRootStore, TClientStore, TReactChangeEvent, TAccount, TAccountsList, TCryptoTransactionDetails } from 'Types';
+import { TReactChangeEvent, TAccount, TAccountsList } from 'Types';
 import CryptoFiatConverter from 'Components/crypto-fiat-converter';
 import ErrorDialog from 'Components/error-dialog';
 import PercentageSelector from 'Components/percentage-selector';
 import RecentTransaction from 'Components/recent-transaction';
 import AccountTransferNote from './account-transfer-form-side-note';
 import SideNote from 'Components/side-note';
 import './account-transfer-form.scss';
-
-type TSelect = {
-    currency: string;
-    balance: number;
-    is_dxtrade: boolean;
-    is_crypto: boolean;
-    is_mt: boolean;
-    value: string;
-    error: string;
-    status?: string;
-};
+import { useStore } from '../../../hooks';
 
 type TAccountTransferFormProps = {
-    account_limits: TClientStore['account_limits'];
-    account_transfer_amount: string;
-    accounts_list: Array<TAccount>;
-    authentication_status: TClientStore['authentication_status'];
-    converter_from_amount: string;
-    converter_from_error: string;
-    converter_to_amount: string;
-    converter_to_error: string;
-    crypto_transactions: Array<TCryptoTransactionDetails>;
     error: object;
-    is_crypto: boolean;
-    is_dxtrade_allowed: TClientStore['is_dxtrade_allowed'];
-    minimum_fee: string;
-    onChangeConverterFromAmount: () => void;
-    onChangeConverterToAmount: () => void;
-    onChangeTransferFrom: (event: TReactChangeEvent) => void;
-    onChangeTransferTo: (event: TReactChangeEvent) => void;
-    onMount: TClientStore['getLimits'];
-    percentage: number;
-    recentTransactionOnMount: () => void;
-    requestTransferBetweenAccounts: ({ amount }: { amount: number }) => void;
-    resetConverter: () => void;
-    selected_from: TSelect;
-    selected_to: TSelect;
-    setAccountTransferAmount: (amount: string) => void;
-    setErrorMessage: (message: string) => void;
     setSideNotes: (notes: Array<string | JSX.Element | JSX.Element[]> | null) => void;
```

Comment:
could we use `setSideNotes: (notes: TSideNotesProps) => void;` as you implement in account-transfer component to be consistent ?